### PR TITLE
Added Login/ post functions

### PR DESCRIPTION
**Pull request by Bereketmebrahtu**

### CODE REVIEW

File: routes/login.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const layout = require('../layout');
+
+const get = (req, res) => {
```
Comment (lascellesabercrombie) [inline routes/login.js:3]: maybe have this as:

 function (req, res) {...} 

just for consistency?

File: routes/login.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    .then((verification) => {
+        if(verification === false) { 
+            throw new Error();
+
+        } else {
+            return auth.createSession(verification);
+        }
+    })
+     .then((sid) => {
+         res.cookie("sis, sid, auth.COOKIE_OPTION");
```
Comment (lascellesabercrombie) [inline routes/login.js:29]: is 'sis' a typo?

File: routes/login.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+         res.redirect("/");
+
+     })
+     .catch(()=> {
+res.status(401).send(layout(`Error`, `<h1 class="error-message"> SOmething went wrong</h1></h1>`))
```
Comment (lascellesabercrombie) [inline routes/login.js:34]: *Something

File: routes/login.js (hunk 2 above)
Comment (joe-dev-public) [reply routes/login.js:29]: The ``.cookie`` method is expecting three arguments here:

1. The cookie name (aka the "key"): this could be anything, but we'll need to read it later, and it makes sense for it to be called ``"sid"``
2. The cookie data (aka the "value"). This needs to be whatever variable name holds the random sid string (confusingly, also ``sid``, but without quotes this time).
3. Cookie options. These will eventually be held by ``COOKIE_OPTION`` in ``auth.js``.

Long story short, you just need to move the second double quote :)

**Comment by lascellesabercrombie** [comment]
Looking good!
